docs/content/build.md

```diff
@@ -26,7 +26,11 @@ To build the worker configs, you can either:
 
 > **It’s finished building when:** there is a message stating `'spatial build build-config' succeeded`.
 
-<%(Callout type="alert" message="If you encounter build errors, you might not have Linux build support enabled. You need Linux build support because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux. To fix this, run the Unity installer and select the build options you are missing.")%>
+<%(Callout type="alert" message="If you encounter build errors, you might not have selected the build supports your game needs during your Unity setup. <br/><br/>
+* You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.<br/> <br/>
+* You need **Mac** build support if you are developing on a Windows PC.<br/>
+* You need **Windows** build support if you are developing on a Mac. <br/>
+This is so your game supports end-users who may be playing on a different platform to your development machine.")%>
```

Comment:
Not sure if I've understood it correctly, but couldn't you simplify this to "You need Linux, Mac, and Windows build support, whatever platform you're developing on"? (This is assuming that you need the build support for the platform you're developing on, as well as the other two?)